catalog/core/catalog-core-urlresourcereader/src/test/java/ddf/catalog/resource/impl/URLResourceReaderTest.java

```diff
@@ -794,17 +821,28 @@ private ResourceResponse doVerification(
     assertThat(name, is(filename));
     assertTrue(resource.getMimeType().toString().contains(expectedMimeType));
 
-    return resourceResponse;
+    assertThat(
+        "The length of the resource in the response should be " + expectedResponseResourceLength,
+        resourceResponse.getResource().getByteArray().length,
+        is(expectedResponseResourceLength));
+
+    assertThat(
+        "The web client should be created with uri=" + uri,
```

Comment:
✏️ This is fine as is so don't feel you need to change it, but it shouldn't technically be necessary to put what the expected uri is equal to in the reason, since the testing framework will list what the actual and expected value was already. EG, when it fails it look will look like this as right now:

`[ERROR]   URLResourceReaderTest.testUnquotedNameInContentDisposition:549->verifyFileFromURLResourceReader:829 The web client should be created with uri=http://127.0.0.1/src/test/resources/data/mydata?uri=63f30ff4dc85436ea507fceeb1396940_blahblahblah&this=that`
`Expected: "http://127.0.0.1/src/test/resources/data/mydata?uri=63f30ff4dc85436ea507fceeb1396940_blahblahblah&this=thatbla"`
 `    but: was "http://127.0.0.1/src/test/resources/data/mydata?uri=63f30ff4dc85436ea507fceeb1396940_blahblahblah&this=that"`